### DISCUSSION TITLE
Release date for Keto's Bulk permissions check functionality?

### DISCUSSION THREAD

**Discussion by evgeniradev** [ory/keto#1606] [Q&A] [answered]
Hi all,

In regards to this announcement https://changelog.ory.sh/announcements/bulk-permission-check-api?utm_source=LaunchNotes&utm_medium=rss , do we know when the new Bulk permissions check Keto functionality will be released? Currently, it looks like it is only available on the master branch.

**Comment by vinckr** [comment, marked as answer]
Hello @evgeniradev 

At the moment it is not possible to commit to a release date or predict when a release is needed, so next release is coming "when its done".
The goal is to release quarterly or at least several times per year.

**Reply by evgeniradev** [reply]
Thank you.
